app/services/submission_service.rb

```diff
@@ -93,8 +93,7 @@ def update_submission(
             reject_non_target_supply_artist(submission.artist_id)
           )
           if !submission.draft? && submission.user && !access_token.nil? &&
-               !submission.my_collection_artwork_id &&
-               submission.user&.save_submission_to_my_collection?
```

Comment:
**thought:** How about we keep this check and instead set the flag value to true for everyone - once we release and everything is fine, we can remove the flag. It's also fine if we remove this as you did since we can redeploy convection as well if something goes wrong

Reply:
I think removing it should be fine. We can always revert this PR.

Reply:
Sure, that sounds good